deployment/terraform/iam.tf

```diff
@@ -99,6 +96,12 @@ resource "aws_iam_role" "container_instance_ec2" {
   assume_role_policy = "${data.aws_iam_policy_document.container_instance_ec2_assume_role.json}"
 }
 
+resource "aws_iam_role_policy" "ec2_ses_send_email" {
+  name = "EC2SESSendEmail"
```

Comment:
I think that it would be good to inject `var.environment` into this name so that the staging and production environments don't collide when inside the same account.